package.json

```diff
@@ -57,5 +57,9 @@
     "overrides": {
       "babel-preset-solid": "workspace:*"
     }
-  }
+  },
+  "engines": {
+    "pnpm": ">=8.6.1"
+  },
+  "packageManager": "pnpm@8.6.1"
```

Comment:
https://nodejs.org/docs/latest-v18.x/api/all.html#all_packages_packagemanager

Until the above docs or [StackOverflow](https://stackoverflow.com/q/71747609) comes to a clear answer/recommendation, I lean conservative and don't think we should specify a PNPM version range.